draft-ietf-quic-transport.md

```diff
@@ -3035,17 +3035,17 @@ packets (except for PMTU probe packets) SHOULD be sized to fit within the
 maximum packet size to avoid the packet being fragmented or dropped
 {{?RFC8085}}.
 
-To optimize capacity efficiency, endpoints SHOULD use Datagram Packetization
-Layer PMTU Discovery ({{!DPLPMTUD=I-D.ietf-tsvwg-datagram-plpmtud}}), or
-implement Path MTU Discovery (PMTUD) {{!RFC1191}} {{!RFC8201}} to determine
-whether the path to a destination will support its desired message size without
-fragmentation.
-
-In the absence of these mechanisms, QUIC endpoints SHOULD NOT send IP packets
-larger than 1280 bytes. Assuming the minimum IP header size, this results in a
-QUIC maximum packet size of 1232 bytes for IPv6 and 1252 bytes for IPv4. A QUIC
-implementation MAY be more conservative in computing the QUIC maximum packet
-size to allow for unknown tunnel overheads or IP header options/extensions.
+An endpoint SHOULD use Datagram Packetization Layer PMTU Discovery
+({{!DPLPMTUD=I-D.ietf-tsvwg-datagram-plpmtud}}) or implement Path MTU Discovery
+(PMTUD) {{!RFC1191}} {{!RFC8201}} to determine whether the path to a destination
+will support a desired message size without fragmentation.
+
+In the absence of these aforementioned mechanisms, QUIC endpoints SHOULD NOT
```

Comment:
"these aforementioned" seems redundant.  Either "these mechanisms" (the original) or "the aforementioned mechanisms," though that sounds over-formal.

Reply:
Yeah, something wasn't sitting right with me after I wrote that out. I'll leave it as it was.